framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -1133,11 +1136,14 @@ void VulkanReplayConsumerBase::CheckResult(const char* func_name, VkResult origi
             // Raise a fatal error if replay produced an error that did not occur during capture.  Format not supported
             // errors are not treated as fatal, but will be reported as warnings below, allowing the replay to attempt
             // to continue for the case where an application may have queried for formats that it did not use.
-            GFXRECON_LOG_FATAL("API call %s returned error value %s that does not match the result from the "
-                               "capture file: %s.  Replay cannot continue.",
-                               func_name,
-                               util::ToString<VkResult>(replay).c_str(),
-                               util::ToString<VkResult>(original).c_str());
+            GFXRECON_LOG_FATAL(
+                "API call at index: %d thread: %d %s returned error value %s that does not match the result from the "
+                "capture file: %s.  Replay cannot continue.",
```

Comment:
Would you mind fixing the double space after the period here (to ` %s. Replay cannot`) since you are touching the line anyway?